lib/system/EventProperty.cpp

```diff
@@ -4,7 +4,7 @@
 //
 #include "EventProperty.hpp"
 
-#include "utils/Utils.hpp"
+#include "utils/StringUtils.hpp"
```

Comment:
From the loops, this looks like this needs `utils/annex_k.h` as well, I'll add it.